docs/streams/stream-basics/consumers.md

```diff
@@ -16,12 +16,12 @@ Messages can be received from streams either synchronously (sync) or asynchronou
 
 ## Acknowledgement (ack)
 
-When a consumer has successfully processed a message, it needs to send an acknowledgement to the Global Data Network (GDN) so that the GDN can discard the message. If no acknowledgement is received, then the GDN stores the message based on [retention and expiry rules](messages.md#message-retention-and-expiry).
+When a consumer has successfully processed a message, it needs to send an acknowledgement to the Global Data Network (GDN) so that the GDN can discard the message. If no acknowledgement is received, then the GDN stores the message based on [retention and expiry rules](messages#message-retention-and-expiry).
```

Comment:
```suggestion
When a consumer has successfully processed a message, it must send an acknowledgement to the Global Data Network (GDN) so that the GDN can discard the message. If no acknowledgement is received, then the GDN stores the message based on [retention and expiry rules](messages#message-retention-and-expiry).
```

Reply:
@jamesm-macrometa, didn't you change the bookmark heading this is aimed at?